### ISSUE TITLE
[Bug] Storm Forge Armor GUI Broken

### ISSUE THREAD

**Issue by Mohron** [ShetiPhian/Localizations-and-Issues#75]
terraqueous-1.11.0-1.3.10
shetiphiancore-1.11.0-3.4.4

An image says 1000 words:

![image](https://cloud.githubusercontent.com/assets/3623122/26286960/8f3511fe-3e36-11e7-8bed-1cee28456a1e.png)
![image](https://cloud.githubusercontent.com/assets/3623122/26286962/9bacfa64-3e36-11e7-9c16-4e2911876e06.png)

Also, is this the only place to contact you as far as support/requests?

**Comment by ShetiPhian** [comment]
Yep that's not right, looks to be an id shift when the offhand slot was added.
I should add access to the offhand slot also.

For bug reports and feature requests open a new issue here.
Mod specific questions are posted on the mods project page.
You'll need to use the CurseForge one, I disabled the Curse comment section to keep all question in one place. (despite Curse using the CurseForge data it provides it's own comment section)

Here is a link to all my projects:
https://minecraft.curseforge.com/members/ShetiPhian/projects
